docs/setup.rst

```diff
@@ -0,0 +1,99 @@
+Install the module in Odoo
+==========================
```

Comment:
This section is not specific to this repo. Maybe this should become a page on the OCA website where we can point to?

Reply:
Either this or we add it in the README.md
@sbidoul